spec/config-spec.coffee

```diff
@@ -1287,6 +1290,54 @@ describe "Config", ->
           atom.config.set 'foo.bar', ['2', '3', '4']
           expect(atom.config.get('foo.bar')).toEqual  [2, 3, 4]
 
+      describe 'when the value has a "color" type', ->
+        beforeEach ->
+          schema =
+            type: 'color'
+            default: 'white'
+          atom.config.setSchema('foo.bar.aColor', schema)
+
+        it 'coerces various types to a color object', ->
+          atom.config.set('foo.bar.aColor', 'red')
+          expect(atom.config.get('foo.bar.aColor')).toEqual {red: 255, green: 0, blue: 0, alpha: 1}
```

Comment:
I imagine the 99% case is that someone will want conversion to hex or rgba string. As a consumer of the api, it sucks that you need to manually create the string to actually use it.

Can provide an object that has a `toString` method? Is that a stick in the spokes for serialization. I think you could define a `toJSON` method on that object to get around any issues.